src/main/java/by/onliner/page/ShoppingCartPage.java

```diff
@@ -0,0 +1,59 @@
+package by.onliner.page;
+
+import by.onliner.util.MoveTo;
+import by.onliner.util.Waiter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ShoppingCartPage extends AbstractPage {
+
+    @FindBy(xpath = "//div[contains(@class, 'cart-form__title')]")
+    WebElement openedShoppingCartMessage;
```

Comment:
please use modifier for all the elements

Reply:
Fixed